translator/rules.go

```diff
@@ -86,15 +86,16 @@ const (
 type Tip string
 
 const (
-	TipPtrSRef    Tip = "sref"
-	TipPtrRef     Tip = "ref"
-	TipPtrArr     Tip = "arr"
-	TipPtrInst    Tip = "inst"
-	TipMemRaw     Tip = "raw"
-	TipTypeNamed  Tip = "named"
-	TipTypePlain  Tip = "plain"
-	TipTypeString Tip = "string"
-	NoTip         Tip = ""
+	TipPtrSRef      Tip = "sref"
+	TipPtrRef       Tip = "ref"
+	TipPtrArr       Tip = "arr"
+	TipPtrInst      Tip = "inst"
+	TipMemRaw       Tip = "raw"
+	TipTypeNamed    Tip = "named"
+	TipTypePlain    Tip = "plain"
+	TipTypeString   Tip = "string"
+	TipTypeUnsigned Tip = "unsigned"
```

Comment:
Note: the only thing added here was the `unsigned` type tip. Everything else is just a realignment.